### ISSUE TITLE
✍️ Contribution period: Spyke Lionel

### ISSUE THREAD

**Issue by spykelionel** [ersilia-os/ersilia#868]
### Week 1 - Get to know the community

- [X] Join the communication channels
- [X] Open a GitHub issue (this one!)
- [ ] Install the Ersilia Model Hub and test the simplest model
- [ ] Write a motivation statement to work at Ersilia
- [ ] Submit your first contribution to the Outreachy site

### Week 2 - Install and run an ML model

- [ ] Select a model from the suggested list
- [ ] Install the model in your system
- [ ] Run predictions for the EML
- [ ] Compare results with the Ersilia Model Hub implementation!
- [ ] Install and run Docker!

### Week 3 - Propose new models

- [ ] Suggest a new model and document it (1)
- [ ] Suggest a new model and document it (2)
- [ ] Suggest a new model and document it (3)

### Week 4 - Prepare your final application

- [ ] Submit the final application in the Outreachy website

**Comment by DhanshreeA** [comment]
Hi @spykelionel welcome to Ersilia. Please let us know if you need any support in completing the rest of the tasks.

**Comment by spykelionel** [comment]
> Hi @spykelionel welcome to Ersilia. Please let us know if you need any support in completing the rest of the tasks.

Thanks @DhanshreeA. I am still trying to find my way around. If I get blocked I will let you know. For now, I am trying to setup **Docker** to work on my **WSL2 (Ubuntu 22.04)**

**Comment by afanyulionel** [comment]
> Hi @spykelionel welcome to Ersilia. Please let us know if you need any support in completing the rest of the tasks.

Fetching my first model from Ersilia model hub takes too long. I have finally succeeded in setting up my environment. I have tried fetching my first model, following the Ersilia book at [ersilia-model-hub](https://ersilia.gitbook.io/ersilia-book/ersilia-model-hub). It's being running for a while now, more than 30 min. Just want to confirm it's a normal behavior and also to find out how long it takes to fetch a model, and what's the required bandwidth. The model in question is the **retrosynthetic-accessibility** or **eos2r5amodel**
